subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoTaskExtension.java

```diff
@@ -52,11 +54,12 @@ public String getAsArg() {
         }
     }
 
+    private Project project;
```

Comment:
We should avoid a pattern where everyone has to inject `Project` everywhere. We want to continue to detangle things from project.